hack/test-integration.sh

```diff
@@ -86,6 +86,7 @@ function exectest() {
 
 	if [[ ${result} -eq 0 ]]; then
 		os::text::print_green "ok      $1"
+		rm -rf /tmp/etcd?????????
```

Comment:
can't assume $TMPDIR points to /tmp, and I'd rather see us choosing the data dir in the script and removing it than assuming we know the magic number of random characters